template/theme/sections/brands-listing.vue

```diff
@@ -141,7 +141,7 @@
                       v-show="
                         checkGlide(entry, settings.props.items_per_row.value)
                       "
-                      v-for="(entry, index) in glidePaginate(
+                      v-for="(entry, ) in glidePaginate(
```

Comment:
Why is this line changed?

Reply:
index use nhi hua tha to error fek rha tha